modules/images-configuration-registry-mirror.adoc

```diff
@@ -191,12 +191,12 @@ unqualified-search-registries = ["registry.access.redhat.com", "docker.io"]
     insecure = false
```

Comment:
In step 5d, could you help update the cmd `sh-4.2# cat /etc/containers/registries` to `sh-4.2# cat /etc/containers/registries.conf`

Reply:
This change has been made and pushed to this branch. Please verify that it is correct.